noel/setrun.py

```diff
@@ -0,0 +1,484 @@
+# encoding: utf-8
+"""
+Module to set up run time parameters for Clawpack.
+
+The values set in the function setrun are then written out to data files
+that will be read in by the Fortran code.
+
+"""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import datetime
+import shutil
+import gzip
+
+import numpy as np
+
+from clawpack.geoclaw.surge.storm import Storm
+import clawpack.clawutil as clawutil
+from clawpack.geoclaw import topotools
+
+
+# Time Conversions
+def days2seconds(days):
+    return days * 60.0**2 * 24.0
+
+
+# Scratch directory for storing topo and dtopo files:
+CLAW = os.environ['CLAW']
+
+scratch_dir = os.path.join(os.environ["CLAW"], 'geoclaw', 'scratch')
+datadir = os.path.join(CLAW,'geoclaw','noel') # directory with noel topo data
```

Comment:
For the apps repository we generally do not use the scratch directory but instead either look for a `DATADIR` environment variable or default to the local directory.